scipy/stats/_continuous_distns.py

```diff
@@ -5831,6 +5832,21 @@ def _stats(self, c):
         excess_kurtosis = sc.polygamma(3, c) / (var*var)
         return mean, var, skewness, excess_kurtosis
 
+    def _entropy(self, c):
+        def regular(c):
+            h = sc.gammaln(c) - c * sc.digamma(c) + c
+            return h
+
+        def asymptotic(c):
+            # using asymptotic expansions for gammaln and psi (see gh-18093)
+            term_1 = -0.5 * np.log(c) + (1 / 6) * c ** -1.
+            term_2 = -(1 / 90) * c ** -3. + (1 / 210) * c ** -5.
+            h = norm._entropy() + term_1 + term_2
```

Comment:
Might as well use the same style as above and simplify.
```suggestion
            term = -0.5*np.log(c) + c**-1./6 - c**-3./90 + c**-5./210
            h = norm._entropy() + term
```